### DISCUSSION TITLE
Pdf template for service proposal

### DISCUSSION THREAD

**Discussion by krseguros** [YetiForceCompany/YetiForceCRM#16773] [How to] [answered]
Hello,

I need to have a pdf template similar to the one I show you in the image, with the intention of being able to use it as a service proposal to prospects and potential customers.

Any manual or instructions on how to achieve something similar with the current version of YetiForceCRM?

Is it possible to develop a template for proposing services to prospects and potential clients with the cu
rrent version of YetiForceCRM?

![image](https://user-images.githubusercontent.com/96985476/210098559-7cc87e68-7679-4e36-9bf3-0d35b989884d.png)

[modelo propuesta de servicios 1NOW-Case-Study.pdf](https://github.com/YetiForceCompany/YetiForceCRM/files/10325736/modelo.propuesta.de.servicios.1NOW-Case-Study.pdf)

**Comment by apcloic** [comment, marked as answer]
@krseguros 
You can generate PDF and build custom templates from any module.
Regarding custom layouts, you will find CSS rules supported by YetiforcePDF here :
https://github.com/YetiForceCompany/YetiForcePDF

If you find it too limited, you may consider subscribing to YetiforcePDF Premium which uses Chromium PDF engine :
https://yetiforce.com/en/yetiforce-pdf-premium-en

Regards,
